javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -68,13 +63,13 @@ public static void clear() {
     // **********************************************************************
 
     /** @return the fully-qualified name of an annotation as a String */
-    public static final /*@Interned*/ String annotationName(AnnotationMirror annotation) {
+    public static final String annotationName(AnnotationMirror annotation) {
         if (annotation instanceof AnnotationBuilder.CheckerFrameworkAnnotationMirror) {
             return ((AnnotationBuilder.CheckerFrameworkAnnotationMirror) annotation).annotationName;
         }
         final DeclaredType annoType = annotation.getAnnotationType();
         final TypeElement elm = (TypeElement) annoType.asElement();
-        /*@Interned*/ String name = elm.getQualifiedName().toString().intern();
+        String name = elm.getQualifiedName().toString();
```

Comment:
This is an example for my first comment: let's change the return type of this method to `Name` and directly return `elm.getQualifiedName()`.

Reply:
I tried this, but it didn't help with performance.